src/tools/gcc.lua

```diff
@@ -284,12 +284,20 @@
 -- Decorate include file search paths for the GCC command line.
 --
 
-	function gcc.getincludedirs(cfg, dirs, sysdirs)
+	function gcc.getincludedirs(cfg, dirs, sysdirs, frameworkdirs)
 		local result = {}
 		for _, dir in ipairs(dirs) do
 			dir = project.getrelative(cfg.project, dir)
 			table.insert(result, '-I' .. p.quoted(dir))
 		end
+
+		if (table.contains(os.getSystemTags(cfg.system), "darwin") and frameworkdirs ~= nil) then
+			for _, dir in ipairs(frameworkdirs or {}) do
```

Comment:
Below doesn't have `or {}`.

Reply:
`and frameworkdirs ~= nil` is probably the one to remove here

Reply:
I believe this one has been resolved?